projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1884,7 +1869,7 @@ protected static function build_sprintf( $args ) {
 	 * @return string
 	 */
 	public function publicize_connections_url( $source = 'calypso-marketing-connections' ) {
-		if ( $this->use_admin_ui_v1() && current_user_can( 'manage_options' ) ) {
+		if ( current_user_can( 'manage_options' ) ) {
```

Comment:
@pablinos I think we should change this check to `publish_posts` like we did for Social admin page.

Reply:
Yeah makes sense. Feel free to make a task/PR for it. I was sure there was something like `publicize_permission` defined somewhere, which if there is we should use, but I can't find it.

Reply:
It's this method but it's available only if Publicize module is ON.

https://github.com/Automattic/jetpack/blob/cb54f2a92c5ddc0396767543feac39a224fafaed/projects/packages/publicize/src/class-publicize-base.php#L1070

Reply:
Oh well, so is this method here. I think that should work.

Reply:
Updated in #42572